plugins/orchestrator/src/utils/formState.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  JsonArray,
+  JsonObject,
+  JsonPrimitive,
+  JsonValue,
+} from '@backstage/types';
+
+export function isJsonPrimitive(value: JsonValue): value is JsonPrimitive {
```

Comment:
you can remove this

Reply:
This fits better inside this module: `src/utils/TypeGuards.ts`

Reply:
BTW, `object` and `array` should also be considered valid JSON values ([source](https://datatracker.ietf.org/doc/html/rfc7159#page-4:~:text=value%20%3D%20false%20/%20null%20/%20true%20/%20object%20/%20array%20/%20number%20/%20string))

Reply:
IMO `TypeGuards` is too generic to have these functions, which are created from [these types](https://github.com/backstage/backstage/blob/master/packages/types/src/json.ts).

Reply:
@batzionb I've removed the unused functions in https://github.com/caponetto/backstage-plugins/pull/66/commits/a9ff51a4f0b088968bdba24e124f9380c968c8d2. Feel free to delete `formState.ts` if the functions are no longer necessary for the multi-step form.